src/overlays/actors/ovl_En_Horse_Ganon/z_en_horse_ganon.h

```diff
@@ -3,10 +3,25 @@
 
 #include <ultra64.h>
 #include <global.h>
+#include <z64.h>
+#include <z64collision_check.h>
```

Comment:
z64collision_check.h is included in z64.h, and z64.h is included in global.h. you dont need either of these lines